report_csv/readme/USAGE.rst

```diff
@@ -36,3 +36,25 @@ A report XML record ::
         file="res_partner"
         attachment_use="False"
     />
+
+There are two options for adding encoding option:
+
+Option 1:
+    - Go to Setting > Technical > reports
+    - Select the report you created with report_type "csv"
+    - Add encoding method in encoding field (eg. cp932)
+
+Option 2:
+    - Add in report XML record
+
+eg.
+    <report
+        id="partner_csv"
+        model="res.partner"
+        string="Print to CSV"
+        report_type="csv"
+        encoding="cp932"
+        name="module_name.report_name"
+        file="res_partner"
+        attachment_use="False"
+    />
```

Comment:
I suppose the report tag doesn't take the encoding parameter.  Did you try this yourself?

We may simply add the following comment instead of a lengthy explanation:

Update `encoding` with an appropriate value (e.g. `cp932`) as necessary.

Reply:
Sorry. You are right. We can only pass value to encoding when we create record with "ir.actions.report".